### ISSUE TITLE
memory leak

### ISSUE THREAD

**Issue by Mamba24L8** [saferwall/pe#35]
https://github.com/saferwall/pe/blob/main/file.go：

should：

```
// Close closes the File.
func (pe *File) Close() error {
	if pe.data != nil {
		_ = pe.data.Unmap()
	}

	if pe.f != nil {
		return pe.f.Close()
	}
	return nil
}
```

**Comment by LordNoteworthy** [comment]
Thanks a lot @Mamba24L8 

That's a good catch, I just submitted a fix, new tag v1.2.2 contains the fixed memory leak.

**Comment by Mamba24L8** [comment]
This is an automatic reply, confirming that your e-mail was received.Thank you
